### PR TITLE
[docs] Update react-native-appearance module documentation with a full example

### PR DESCRIPTION
**Pull request by rmonaghanjr**
# Why

The current documentation on the expo website does not show the full documentation for the module, this Pull Request fixed that.

# How

I used the help that u/brentvatne gave to me on reddit, and added them into the documentation page. I also added a full example of an app that supports Light/Dark system themes.

# Test Plan

I tested this change by looking at the markdown and concluding that it looks like official documentation. I also tested the example app and it works as intended on my iDevice.

### CODE REVIEW

**Comment by ide** [comment]
@cruzach could you please do a pass over this when you get back (rewrite as necessary) so that this is consistent with the other docs?

**Comment by brentvatne** [comment]
hi there! I just noticed this PR, I have updated the documentation already to clarify the app.json key: https://docs.expo.io/versions/v35.0.0/sdk/appearance/#configuration

do you think this is sufficient?

**Comment by AEJester** [comment]
> hi there! I just noticed this PR, I have updated the documentation already to clarify the app.json key: https://docs.expo.io/versions/v35.0.0/sdk/appearance/#configuration
> 
> do you think this is sufficient?

I do think what you have added is satisfactory, but having an example in the documentation until you can update snack to support ios13 (has it already?) would be even better. Please change what you need in my PR (or just don't merge it lol, it's up to you guys), and thanks for all the help you've given me!

File: docs/pages/versions/v35.0.0/sdk/appearance.md [hunk 1]
```diff
@@ -53,4 +54,41 @@ function MyComponent() {
     // render some light thing
   }
 }
-```
\ No newline at end of file
+```
+-----
+Full example:
+```js
+import react, {Component} from 'react';
```
Comment (EvanBacon) [inline docs/pages/versions/v35.0.0/sdk/appearance.md]:
```suggestion
import React, {Component} from 'react';
```

File: docs/pages/versions/v35.0.0/sdk/appearance.md [hunk 2]
```diff
@@ -53,4 +54,41 @@ function MyComponent() {
...
+import {Text, View} from 'react-native';
+
+//import the modules
+import {Appearance, AppearanceProvider} from 'react-native-appearance';
+
+export default class App extends Component {
```
Comment (EvanBacon) [inline docs/pages/versions/v35.0.0/sdk/appearance.md]: Rewrite with hooks plx

**Comment by brentvatne** [comment]
an example could be useful! as @EvanBacon's advice here would be good to follow if you're interested in updating it

File: docs/pages/versions/v35.0.0/sdk/appearance.md [hunk 3]
```diff
@@ -53,4 +54,41 @@ function MyComponent() {
...
+    }
+    this.subscription = Appearance.addChangeListener(({ colorScheme }) => { //Add subscription to when the user changes themes
+    //set the theme in state so that the app re-renders for the theme
+      this.setState({theme: colorScheme})
+    })
+  }
+  render() {
+    return (
+      <AppearanceProvider>
+        <View stle={{
```
Comment (cruzach) [inline docs/pages/versions/v35.0.0/sdk/appearance.md]:
```suggestion
        <View style={{
```

**Review by cruzach — CHANGES_REQUESTED** [review]
along with Evan's requested changes, just copy/paste this into https://snack.expo.io/ and hit `Prettier {}` so the formatting fits the rest of our examples/snacks

**Review by EvanBacon — APPROVED** [review]
LGTM thanks for the hooks :D

**Pull request merged**